md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessAPI.java

```diff
@@ -659,10 +732,9 @@ RestWrapper importData(@ModelAttribute("fileString")
                         daoProcessTemplate.setProcessTemplateId(process.getProcessTemplateId());
                         insertDaoProcess.setProcessTemplate(daoProcessTemplate);
                     }
-                    if (dbParentProcess!= null) {
+                    if (dbParentProcess != null) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)